app/javascript/pages/country/components/widget-tree-cover-loss-areas/widget-tree-cover-loss-areas-component.js

```diff
@@ -0,0 +1,84 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { PieChart, Pie, Legend, Tooltip, Cell }from 'recharts';
+import numeral from 'numeral';
+
+import TooltipChart from '../tooltip-chart/tooltip-chart';
+import WidgetHeader from '../widget-header/widget-header';
+
+class WidgetTreeLossAreas extends PureComponent {
+  componentDidMount() {
+    const { setInitialData } = this.props;
+    setInitialData(this.props);
+  }
+
+  render() {
+    const {
+      isLoading,
+      countryData,
+    } = this.props;
+
+    const pieCharDataDistricts = [
```

Comment:
It's better if you set this hardcode array inside redux! In this way, you don't need to touch the pure component again when you get the endpoint.